loadrunner_professional/assets/logs/loadrunner_professional.yaml

```diff
@@ -0,0 +1,62 @@
+id: loadrunner
+metric_id: loadrunner-professional
+backend_only: false
+facets: null
+pipeline:
+  type: pipeline
+  name: LoadRunner Scenario Information Pipeline
+  enabled: true
+  filter:
+    query: "source:loadrunner"
+  processors:
+    - type: attribute-remapper
+      name: Map attribute `hostname` to tag `host`
```

Comment:
I don't see the `hostname` attribute in any of the samples. Is it because it's provided outside of the message field or because we're missing a representative sample ?

Reply:
Indeed, it is provided outside the message field.